### PR TITLE
ME-1893: Add connector auth flag in socket config

### PR DESCRIPTION
**Pull request by waltzofpearls**

### CODE REVIEW

File: connector/connector.proto [hunk 1]
```diff
@@ -121,7 +121,8 @@ message SocketConfig {
   string id = 1;
   string name = 2;
   string type = 3;
-  google.protobuf.Struct config = 4;
+  bool connector_authentication_enabled = 4;
```
Comment (adrianosela) [inline connector/connector.proto:124]: Do we need this? I thought we dont support disabling this anymore?

**Review by adrianosela — CHANGES_REQUESTED** [review]

**Review by adrianosela — APPROVED** [review]

File: connector/connector.proto (hunk 1 above)
Comment (btoonk) [reply connector/connector.proto:124]: This is a breaking change, if we want this we should not change the ordering.
But I think it's better to just make this part of the config struct and not an additional field in the SocketConfig.

**Review by btoonk — CHANGES_REQUESTED** [review]
This is a breaking change, if we want this we should not change the ordering.
But I think it's better to just make this part of the config struct and not an additional field in the SocketConfig.

**Comment by waltzofpearls** [comment]
Closing this PR. Will not add another field, and instead, pass `ConnectorAuthenticationEnabled` in `config`.

**Pull request closed without merging**